### PR TITLE
Improve Javadoc in RsFlash #540

### PR DESCRIPTION
**Pull request by essobedo**

### CODE REVIEW

**Comment by davvd** [comment]
@essobedo Thanks for your pull request, let me find someone who can review it

**Comment by davvd** [comment]
@mkordas please review, thanks

**Comment by mkordas** [comment]
@essobedo I'm on it

File: src/main/java/org/takes/facets/flash/RsFlash.java [hunk 1]
```diff
@@ -40,17 +40,28 @@
...
+ * dynamic content to render in which you want to add success or error
+ * messages. The typical use case is when you have a form that the user can
+ * submit and you want to be able to indicate whether the request was
+ * successful or not.
+ *
+ * <p>The flash mechanism is a stateful mechanism based on a cookie so it is
+ * not meant to be used to implement stateless components such as a RESTful
+ * service.
+ *
+ *
```
Comment (mkordas) [inline src/main/java/org/takes/facets/flash/RsFlash.java]: @essobedo unnecessary double newline, please remove

File: src/main/java/org/takes/facets/flash/RsFlash.java [hunk 2]
```diff
@@ -40,17 +40,28 @@
...
+ * <p>This class helps you to automate the flash message mechanism, by
```
Comment (mkordas) [inline src/main/java/org/takes/facets/flash/RsFlash.java]: @essobedo I think this should stay second sentence.

File: src/main/java/org/takes/facets/flash/RsFlash.java [hunk 3]
```diff
@@ -62,14 +73,10 @@
...
- *       throw new RsFlash(
- *         new RsForward(),
- *         "can't save your post, sorry",
- *         java.util.logging.Level.SEVERE
- *       );
- *     }
+ *   public Response act(final Request req) throws IOException {
+ *     return new RsForward(
+ *         new RsFlash("can't save your post, sorry",
+ *         java.util.logging.Level.SEVERE));
```
Comment (mkordas) [inline src/main/java/org/takes/facets/flash/RsFlash.java]: @essobedo please fix code sample according to http://www.yegor256.com/2014/10/23/paired-brackets-notation.html

**Comment by mkordas** [comment]
@essobedo please see a few comments above

**Comment by mkordas** [comment]
@essobedo please also provide PR description

**Comment by mkordas** [comment]
@essobedo why you've closed this PR? Please continue work here and close #570

**Comment by essobedo** [comment]
@mkordas because my fork was corrupted so I recreated it from scratch so it is not even possible to keep this PR as the corresponding branch doesn't exist anymore so now the PR is https://github.com/yegor256/takes/pull/570

**Comment by mkordas** [comment]
@essobedo how did your fork become corrupted? Please always ask for help before doing such drastic things like removing fork. Usually `push --force` is enough to repair remote repository.

**Comment by essobedo** [comment]
@mkordas I agree with you, it was a mistake, it won't happen again. Please check my new PR which is https://github.com/yegor256/takes/pull/570

**Comment by mkordas** [comment]
@essobedo please at least update PR description with issue number

**Comment by davvd** [comment]
@ypshenychka please, review this ticket for compliance with our [QA rules](http://at.teamed.io/qa.html)

**Comment by essobedo** [comment]
@davvd @ypshenychka Please ignore this PR, it was a mistake on my side, the new PR for this issue is https://github.com/yegor256/takes/pull/570. Sorry for the noise.

**Comment by ypshenychka** [comment]
@essobedo 
According to our [QA Rules](http://at.teamed.io/qa.html):
>Pull request description explains the solution proposed and contains a link to the original ticket it is related to.

Please provide appropriate PR description.

**Comment by essobedo** [comment]
@ypshenychka Please ignore this PR, it was a mistake on my side, the new PR for this issue is https://github.com/yegor256/takes/pull/570. Sorry for the noise.

**Comment by mkordas** [comment]
@essobedo you can't ask to ignore PR when it has reviewer assigned.

**Comment by ypshenychka** [comment]
@essobedo I can't ignore the PR. @mkordas did his job and should be paid for it after I finish review.
So please do appropriate corrections to description.

**Comment by essobedo** [comment]
@ypshenychka As you wish, here is the description

**Comment by ypshenychka** [comment]
@essobedo Thank you

**Comment by ypshenychka** [comment]
@davvd Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@ypshenychka thank you, let's try to make it "good" next time

**Comment by davvd** [comment]
@mkordas 10 mins was added to the account of @ypshenychka (for QA review), in transaction 76643813... Thanks a lot, I just topped your account for **25 mins**, transaction ID `56b7a86cbaeaef2f300002c6`, total time was 22 hours and 30 mins... review comments (c=10) added as a bonus... +25 added to your rating, at the moment it is: [+5432](http://www.netbout.com/b/36080?open=rating)

**Pull request closed without merging**